apps/app/src/app/pages/pages.component.html

```diff
@@ -21,16 +21,27 @@
         routerLinkActive="active"
         >{{ link.label }}</a
       >
-      <div class="nav-divider" *ngIf="isSeqEnabled || isGrafanaEnabled"></div>
-      <a clrVerticalNavLink href="/seq/" target="_blank" *ngIf="isSeqEnabled">
+      <div
```

Comment:
Tu as la possibilité de ne faire le async qu'une fois (sur chacun des services) pour éviter la redondance entre cette ligne et la ligne 35 ou 44.

Je te laisse lire ça dans cette article : https://medium.com/@yurykatkov/how-to-avoid-multiple-async-pipes-in-angular-ff0d51a8d368